src/autoqasm/api.py

```diff
@@ -35,6 +35,63 @@
 from autoqasm.program.gate_calibrations import GateCalibration
 from autoqasm.types import QubitIdentifierType as Qubit
 
+reserved_keywords = [
```

Comment:
Do you have a reference/link to the source of this list?

Reply:
https://openqasm.com/grammar/index.html

Reply:
Thanks, I think in long term, it would be better that this list is not hard-coded. Rather, the list loads the qasm3lexer from the lexer file. But I think it does not necessary need to be implement in this PR. Could you add the [reference link](https://github.com/openqasm/openqasm/blob/main/source/grammar/qasm3Lexer.g4) as an inline comment?

Reply:
> Thanks, I think in long term, it would be better that this list is not hard-coded. Rather, the list depends on the qasm3lexer from the lexer file. But I think it does not necessary need to be implement in this PR. Could you add the [reference link](https://github.com/openqasm/openqasm/blob/main/source/grammar/qasm3Lexer.g4) as an inline comment?

Yes, initially, I had done it that way.